src/python/dxpy/scripts/dx.py

```diff
@@ -5316,6 +5346,8 @@ def positive_integer(value):
 parser_watch.add_argument('-f', '--format', help='Message format. Available fields: job, level, msg, date')
 parser_watch.add_argument('--no-wait', '--no-follow', action='store_false', dest='tail',
                           help='Exit after the first new message is received, instead of waiting for all logs')
+parser_watch.add_argument('--metrics', help=fill('Select display mode for detailed job metrics if they were collected and are available based on retention policy; see --metrics-help for details', width_adjustment=-24),
```

Comment:
will @wormsik add `--metrics top` and `--metrics-help` in a separate commit?

Reply:
Yes